<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Controls</class>
 <widget class="QDockWidget" name="Controls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>641</height>
   </rect>
  </property>
  <property name="allowedAreas">
   <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
  </property>
  <property name="windowTitle">
   <string>Controls</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Add Solver</string>
      </property>
      <property name="buddy">
       <cstring>addSolvCombo</cstring>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="addSolvCombo">
      <item>
       <property name="text">
        <string>None</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1st Order Explicit Euler</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Least Squares</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Simple Implicit</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Finite Element Method</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>LeastSquares2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Runge-Kutta</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="sizeLabel">
      <property name="text">
       <string>Number of Points</string>
      </property>
      <property name="buddy">
       <cstring>intNumberOfPoints</cstring>
      </property>
     </widget>
    </item>
    <item>
     <widget class="KIntNumInput" name="intNumberOfPoints">
      <property name="label">
       <string/>
      </property>
      <property name="value">
       <number>100</number>
      </property>
      <property name="minimum">
       <number>4</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="cyclesLabel">
      <property name="text">
       <string>Number of Cycles</string>
      </property>
      <property name="buddy">
       <cstring>cyclesInput</cstring>
      </property>
     </widget>
    </item>
    <item>
     <widget class="KDoubleNumInput" name="cyclesInput">
      <property name="value">
       <double>1.000000000000000</double>
      </property>
      <property name="minimum">
       <double>-9999.000000000000000</double>
      </property>
      <property name="decimals">
       <number>6</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="cflLabel">
      <property name="text">
       <string>CFL</string>
      </property>
      <property name="buddy">
       <cstring>cflInput</cstring>
      </property>
     </widget>
    </item>
    <item>
     <widget class="KDoubleNumInput" name="cflInput">
      <property name="value">
       <double>1.000000000000000</double>
      </property>
      <property name="minimum">
       <double>-9999.000000000000000</double>
      </property>
      <property name="decimals">
       <number>6</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="stepsLabel">
      <property name="text">
       <string>Time Steps</string>
      </property>
      <property name="buddy">
       <cstring>intTimeSteps</cstring>
      </property>
     </widget>
    </item>
    <item>
     <widget class="KIntNumInput" name="intTimeSteps">
      <property name="value">
       <number>100</number>
      </property>
      <property name="minimum">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="incrementLabel">
      <property name="text">
       <string>Plot Increment</string>
      </property>
      <property name="buddy">
       <cstring>intPlotIncrement</cstring>
      </property>
     </widget>
    </item>
    <item>
     <widget class="KIntNumInput" name="intPlotIncrement">
      <property name="value">
       <number>1</number>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Plot Delay</string>
      </property>
      <property name="buddy">
       <cstring>plotDelayInput</cstring>
      </property>
     </widget>
    </item>
    <item>
     <widget class="KIntNumInput" name="plotDelayInput"/>
    </item>
    <item>
     <widget class="QPushButton" name="savePlotButton">
      <property name="text">
       <string>Save Image</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="animationCheck">
      <property name="text">
       <string>Save Anim. Frames</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox">
      <property name="text">
       <string>Save Error Data</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>237</width>
        <height>125</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="resetButton">
        <property name="text">
         <string>Reset</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="runButton">
        <property name="text">
         <string>Run</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="stopButton">
        <property name="text">
         <string>Stop</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KDoubleNumInput</class>
   <extends>QWidget</extends>
   <header>knuminput.h</header>
  </customwidget>
  <customwidget>
   <class>KIntNumInput</class>
   <extends>QWidget</extends>
   <header>knuminput.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
